### ISSUE TITLE
extra.symfony.require breaks composer.json validity

### ISSUE THREAD

**Issue by fbourigault** [symfony/flex#421]
Since the introduction of the `extra.symfony.require` setting, Symfony dependencies constraints are considered unbound by composer. So running `composer validate --strict` output some warnings and return an non-0 exit code which, in my case, breaks CI composer validation.

To reproduce the issue, just create an new Symfony flex using `composer create-project symfony/skeleton flex-issue`, then fill the `name` and `properties` fields and run `composer validate --strict`

```
./composer.json is valid, but with a few warnings
See https://getcomposer.org/doc/04-schema.md for details on the schema
require.symfony/console : unbound version constraints (*) should be avoided
require.symfony/framework-bundle : unbound version constraints (*) should be avoided
require.symfony/yaml : unbound version constraints (*) should be avoided
```

**Comment by nicolas-grekas** [comment]
We could use `4.*` in the skeletons maybe. But from a maintainer perspective, this will create merge conflicts when merging 3.4 into 4.* (because we'll use `3.*` there of course).

**Comment by nicolas-grekas** [comment]
fixed in #435
